cpp/include/cudf/strings/convert/convert_fixed_point.hpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace cudf {
+namespace strings {
+/**
+ * @addtogroup strings_convert
+ * @{
+ * @file
+ */
+
+/**
+ * @brief Returns a new fixed-point column parsing decimal values from the
+ * provided strings column.
+ *
+ * Any null entries result in corresponding null entries in the output column.
+ *
+ * The expected format is [-+][0-9].[0-9] for each string. An invalid data format
+ * result in undefined behavior in the corresponding output row result.
```

Comment:
The regex isn't strictly correct, and probably should be as a specification... The sign and `.` are optional, and there can be zero or more digits before or after the decimal...  Not sure how precise we want to be here. 

Also a small typo:

```suggestion
 * The expected format is [-+][0-9].[0-9] for each string. An invalid data format
 * results in undefined behavior in the corresponding output row result.
```

Reply:
One option is to specify it like this:

```
The expected format is `[sign][integer][.][fraction]`, where the sign is either not present, `-` or `+`, The decimal point `[.]` may or may not be present, and `integer` and `fraction` are comprised of zero or more digits in [0-9].
```

Reply:
Thanks. This looks good. I'd rather not burden myself or the reader to understand some regex. :-)